### ISSUE TITLE
Update package in npm

### ISSUE THREAD

**Issue by MrFiregore** [silvermine/eslint-plugin-silvermine#59]
Please, update package in npm with the new changes

**Comment by jthomerson** [comment]
@MrFiregore what changes are you looking to get that aren't already in v2.3.1? There's not a lot there, as we're just working through some general package upgrades.

@yokuze any advantage to cutting a version soon?

```
[ master ] $ git log --oneline --no-merges --reverse v2.3.1..
34a7d93 chore: Update TypeScript to v4.6.3 and Grunt
2de07b4 chore: Upgrade to ESLint 8
df0037e chore: Add @silvermine/standardization
899e2b2 chore: Update @silvermine/eslint-config to v3.1.0-beta.1
```

**Comment by yokuze** [comment]
> @MrFiregore what changes are you looking to get that aren't already in v2.3.1? There's not a lot there, as we're just working through some general package upgrades.
> 
> @yokuze any advantage to cutting a version soon?
> 
> ```
> [ master ] $ git log --oneline --no-merges --reverse v2.3.1..
> 34a7d93 chore: Update TypeScript to v4.6.3 and Grunt
> 2de07b4 chore: Upgrade to ESLint 8
> df0037e chore: Add @silvermine/standardization
> 899e2b2 chore: Update @silvermine/eslint-config to v3.1.0-beta.1
> ```

@jthomerson We've been waiting to cut a new release until we are done testing the latest changes from `master`. We are on [step 6 of the dance we do to update ESLint in our config and this plugin](https://github.com/silvermine/eslint-config-silvermine#updating-eslint). We think we are good to cut a new version of the plugin now, though. The new version would be: `v2.4.0`. Thanks!

**Comment by yokuze** [comment]
We've published `v2.4.0`: 

* https://www.npmjs.com/package/@silvermine/eslint-plugin-silvermine/v/2.4.0
* https://github.com/silvermine/eslint-plugin-silvermine/compare/v2.3.1...v2.4.0
